src/patterns/accordion.scss

```diff
@@ -71,3 +71,88 @@
 		}
 	}
 }
+
+.wp-block-wmf-reports-accordion--alt {
```

Comment:
I don't like that this is a magic string. Is it possible to set this as a style variant on the accordion block?

Reply:
We absolutely could @kadamwhite, the previous style override was done with a class, so I wanted to keep that approach consistent.  

I would suggest both report variants having a custom style applying to them.

Are we ok to ticket that up and we can come back to it?

Reply:
@mattwatsoncodes Ticket away! Agree that technically it's not something we need to get done this year, but would be good to clean up in the future.